Library/Formula/mpd.rb

```diff
@@ -116,4 +119,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork { exec "#{bin}/mpd --stdout --no-daemon --no-config" }
+    sleep 2
+    Process.kill("SIGINT", pid)
+    Process.wait(pid)
+  end
```

Comment:
It this a server or similar? If so, can it be tested by `curl`. Also `Process.kill/wait` should be inside a `ensure` block.